cluster/saltbase/salt/kube-controller-manager/kube-controller-manager.manifest

```diff
@@ -19,6 +20,9 @@
 {% if pillar['enable_deployments'] is defined -%}
   {% set enable_deployments = "--enable-deployment-controller=" + pillar['enable_deployments'] -%}
 {% endif -%}
+{% if grains.cloud in [ 'gce' ] and grains.cloud_config is defined -%}
```

Comment:
Changing default SGTM -- fewer special cases where k8s behaves in way X in environment Y but in a different way in Z generally good?  I skimmed #7660 and didn't see a push for GCE-only, but maybe missed something.  Pull in anyone who'd potentially object strongly?